apidemo/views.py

```diff
@@ -8,15 +9,52 @@
 from .categoryModel import Category
 from .userModel import User
 from .articlesModel import Articles
-from rest_framework.permissions import BasePermission ,AllowAny
+import time
+import jwt
+from datetime import datetime,timedelta,timezone
+key="tuan"
```

Comment:
signing `key` should not be hardcoded. Consider to move it to config/environment file